bin/lib/installable/installable.py

```diff
@@ -109,6 +109,9 @@ def should_install(self) -> bool:
         if self.install_context.only_nightly and not self.nightly_like:
             return False
 
+        if self.install_context.only_native_aarch64 and self.config_get("if", "") != "native_aarch64":
```

Comment:
Pragmatic, but I can't help wondering if a more general `--only=native_aarch64` would be easier and more extensible. The `nightly` flag is special as we heuristically guess it rather than it being an actual `if` check in some cases

Reply:
`--only` is not possible, i have experimented with it, but it breaks all the dependency handling and whatnot

Reply:
wouldn't it be exactly the same as what you have there? Like..

```py
if self.install_context.only and self.config_get("if", "") != self.install_context.only:
```

Reply:
Ah I see that way, hmm, maybe. I experimented with making the filter do the work with narrowing the selection, but it's very hard to work with that code in general because the `if` information is not complete there like it is with `config_get()`.

I'll give it a try